exercises/reverse-string/lib/reverse_string.dart

```diff
@@ -1,3 +1,3 @@
-String reverse() {
+class ReverseString {
```

Comment:
This can remain a function instead of a class.

It's part of the process. What's generated is not necessarily the standard. Some exercises are written so a class isn't required if the logic is simple enough to be contained in a function. It also exposes users to the fact that Dart does not require classes.

Reply:
No way to do that, at this time.